cssparse/shared/src/test/scala/cssparse/PrettyPrinterTests.scala

```diff
@@ -0,0 +1,332 @@
+            |    border: 1px solid #ddd;
+            |  }
+            |  .table-responsive > .table {
+            |    margin-bottom: 0;
+            |  }
+            |  .table-responsive > .table > thead > tr > th,
+            |  .table-responsive > .table > tbody > tr > th,
+            |  .table-responsive > .table > tfoot > tr > th,
+            |  .table-responsive > .table > thead > tr > td,
+            |  .table-responsive > .table > tbody > tr > td,
+            |  .table-responsive > .table > tfoot > tr > td {
+            |    white-space: nowrap;
+            |  }
+            |  .table-responsive > .table-bordered {
+            |    border: 0;
+            |  }
+            |  .table-responsive > .table-bordered > thead > tr > th:first-child,
+            |  .table-responsive > .table-bordered > tbody > tr > th:first-child,
+            |  .table-responsive > .table-bordered > tfoot > tr > th:first-child,
+            |  .table-responsive > .table-bordered > thead > tr > td:first-child,
+            |  .table-responsive > .table-bordered > tbody > tr > td:first-child,
+            |  .table-responsive > .table-bordered > tfoot > tr > td:first-child {
+            |    border-left: 0;
+            |  }
+            |  .table-responsive > .table-bordered > thead > tr > th:last-child,
+            |  .table-responsive > .table-bordered > tbody > tr > th:last-child,
+            |  .table-responsive > .table-bordered > tfoot > tr > th:last-child,
+            |  .table-responsive > .table-bordered > thead > tr > td:last-child,
+            |  .table-responsive > .table-bordered > tbody > tr > td:last-child,
+            |  .table-responsive > .table-bordered > tfoot > tr > td:last-child {
+            |    border-right: 0;
+            |  }
+            |  .table-responsive > .table-bordered > tbody > tr:last-child > th,
+            |  .table-responsive > .table-bordered > tfoot > tr:last-child > th,
+            |  .table-responsive > .table-bordered > tbody > tr:last-child > td,
+            |  .table-responsive > .table-bordered > tfoot > tr:last-child > td {
+            |    border-bottom: 0;
+            |  }
+            |}
+            |
+            |@-o-keyframes progress-bar-stripes {
+            |  from {
+            |    background-position: 40px 0;
+            |  }
+            |  to {
+            |    background-position: 0 0;
+            |  }
+            |}
+          """.stripMargin
+
+        val output =
+          """
+            |html{
+            |  font-family : sans-serif ;
+            |  -webkit-text-size-adjust : 100% ;
+            |  -ms-text-size-adjust : 100% ;
+            |}
+            |
+            |body{
+            |  margin : 0 ;
+            |}
+            |
+            |article , aside , details , figcaption , figure , footer , header , hgroup , main , menu , nav , section , summary{
+            |  display : block ;
```

Comment:
Could you tweak the pretty-printer to have no space before each `,`, `:` or `;`?